src/Samples/Toolkit.SampleApp.UWP/Samples/SymbolDisplay/SymbolEditorSample.xaml

```diff
@@ -6,11 +6,9 @@
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
     xmlns:esri="using:Esri.ArcGISRuntime.Toolkit.UI.Controls"
-    mc:Ignorable="d"
-    RequestedTheme="Light">
-
-    <Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+    mc:Ignorable="d">
 
+    <Grid>
```

Comment:
ApplicationPageBackgroundThemeBrush should adjust to the theme. Did this needed to be removed?

Reply:
I tested without this and it looked redundant to apply Background to this Grid as `rootFrame` have the same effect on the look. So I thought of removing it.